src/client/coll.rs

```diff
@@ -0,0 +1,201 @@
+use bson;
+use client::db::Database;
+use client::common::{ReadPreference, WriteConcern};
+use client::wire_protocol::operations::{OpQueryFlags, Message};
+
+/// Interfaces with a MongoDB collection.
+pub struct Collection<'a> {
+    db: &'a Database<'a>,
+    pub name: String,
+    pub full_name: String,
```

Comment:
This should be called 'namespace'. The 'dbname.collname' string is called a namespace in MongoDB parlance.